keyboards/handwired/freoduo/config.h

```diff
@@ -0,0 +1,52 @@
+/*
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x0616 //F is 6th in alphabet, P is 16th :)
```

Comment:
Unfortunately, this VID is already in use. 
https://www.the-sz.com/products/usbid/index.php?v=0x0616

Please change this. ❯ 

For instance: 
```sh
echo -n "ParyzFilip" | shasum | cut -c1-4 | tr '[:lower:]' '[:upper:]'
```
produces: 
```suggestion
#define VENDOR_ID       0xB141
```
Which appears to be unused.